_posts/0000-01-02-BatmanNeuroGuy.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
_The **angle** of the **bedangle** is directly proportional to the **heat** of the **beat**_